oeps/processes/oep-0001.rst

```diff
@@ -245,7 +245,7 @@ OEP Status
         "Under Review" -> { "Deferred" "Provisional" } [dir=both]
         "Under Review" ->  { "Accepted" "Rejected" "Withdrawn" }
         "Accepted" -> "Final"
```

Comment:
Once an OEP is in "Needs Revision", where can it go from there? I haven't tried rendering it, but maybe this is enough? 

```suggestion
        "Accepted" -> "Final"
        "Needs Review" ->  "Final"
```

Reply:
Good point, and, it's tough because (a) Needs Review should just wrap around to Under Review (and all its possible states), and (b) GitHub isn't rendering the graph in preview and I'm having some issue with my local env at the moment.

Reply:
Ha! I forgot we have a build :) I added like this:

<img width="666" alt="image" src="https://github.com/openedx/open-edx-proposals/assets/1985317/c84eb3e9-b857-4cbb-be48-57feadd93ea3">

How does that look? I don't _love_ it but otoh I love how it's autogenerated!

Reply:
Oh yes, autogenerated FTW!

That looks good enough.